crew

```diff
@@ -334,28 +334,31 @@ def remove (pkgName)
     return
   end
 
-  Dir.chdir CREW_CONFIG_PATH do
-
-    #remove all files installed by the package
-    File.open("meta/#{pkgName}.filelist").each_line do |line|
-      begin
-        File.unlink line.chomp
-      rescue => exception #swallow exception
+  #if the package is marked as 'is_fake', the file/directorylists doesn't exist
```

Comment:
Don't think so since we would like to remove files and directories anyway.  Maybe it is possible to write something we've considered it in the comment?  :)